DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/latency_control/src/CMakeLists.txt

```
@@ -28,7 +28,12 @@ endif()
 set(EMULATOR_COMPILE_FLAGS "-Wall ${WIN_FLAG} -fintelfpga -DFPGA_EMULATOR")
 set(EMULATOR_LINK_FLAGS "-fintelfpga")
 set(HARDWARE_COMPILE_FLAGS "-Wall ${WIN_FLAG} -fintelfpga")
-set(HARDWARE_LINK_FLAGS "-fintelfpga -Xshardware -Xsboard=${FPGA_BOARD} ${USER_HARDWARE_FLAGS}")
+if(FPGA_BOARD STREQUAL "intel_a10gx_pac:pac_a10")
```

Please move this above, with the other if/else statements to keep a cluster of compile/link flags 
```suggestion
if(FPGA_BOARD MATCHES ".*a10.*")
```

---

Good point. Will do.